.github/workflows/release.yaml

```diff
@@ -16,7 +16,55 @@ jobs:
     name: Test, lint, & build
     uses: ./.github/workflows/test.yaml
 
+  prerelease:
+    name: Update documentation
+    runs-on: ubuntu-latest
+    outputs:
+      pr_number: ${{ steps.create_pr.outputs.pr_number }}
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v2
+        with:
+          app_id: ${{ secrets.OS_GITHUB_APP_ID }}
+          private_key: ${{ secrets.OS_GITHUB_APP_PRIVATE_KEY }}
+
+      - name: "☁️ checkout repository"
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ steps.generate_token.outputs.token }}
+
+      - name: "🤲 Setup Just"
+        uses: extractions/setup-just@v2
+
+      - name: "📗 generate documentation"
+        run: just gen-docs
```

Comment:
This is missing a "setup go" step:

```yaml
    - name: "🐹 Setup Go"
      uses: actions/setup-go@v5
      with:
        go-version: 1.22.x
```

I don't _think_ the GitHub runners have Go installed, but before running `just gen-docs` (which executes `go run`), I'd go ahead and get Go setup